apps/emqx_management/test/emqx_mgmt_trace_api_SUITE.erl

```diff
@@ -112,6 +114,30 @@ t_http_test(_Config) ->
     unload(),
     ok.
 
+t_download_log(_Config) ->
+    ClientId = <<"client-test-download">>,
+    Now = erlang:system_time(second),
+    Start = to_rfc3339(Now),
+    Name = <<"test_client_id">>,
+    load(),
+    ok = emqx_trace:create([{<<"name">>, Name},
+        {<<"type">>, clientid}, {<<"clientid">>, ClientId}, {<<"start_at">>, Start}]),
+    ct:sleep(50),
```

Comment:
Nit: Use a snabbkaffe trace and wait for it

Reply:
done